### PR TITLE
Add new questions to FAQ

### PR DESCRIPTION
**Pull request by martijnrusschen**
Improving FAQ with commonly asked questions

TODO:
- [x] Add list of variables and examples and description for each

### CODE REVIEW

File: docs/programs/jira-variables.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Jira Variables"
```
Comment (reedloden) [inline docs/programs/jira-variables.md]: Can we call this "Integration Variables" or "Integration Placeholders" and not reference Jira at all?

File: docs/programs/integration-variables.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Integration Variables"
+path: "/programs/integration-variables.html"
+id: "programs/integration-variables"
+---
+
+Variable | Description | Example
+-------- | ------ |
+`{{assignee}}` | The user that's assigned to the report. |
+`{{days_after_report(days)}}` | Provides a date based on the report creation date and the provided arugment on the provided argument. `{{days_after_report(5)}}` will output a date 5 days after the report was created. |
```
Comment (saida) [inline docs/programs/integration-variables.md]:
```suggestion
`{{days_after_report(days)}}` | Provides a date based on the report creation date and the provided argument. `{{days_after_report(5)}}` will output a date 5 days after the report was created. |
```

File: docs/programs/integration-variables.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+`{{details_custom_jira}}` | The link, date, reporter, weaknesss, and description of the vulnerability in the report in a special Jira flavored markdown format. |
```
Comment (saida) [inline docs/programs/integration-variables.md]:
```suggestion
`{{details_custom_jira}}` | The link, date, reporter, weakness, and description of the vulnerability in the report in a special Jira flavored markdown format. |
```

File: docs/programs/integration-variables.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+`{{details_markdown}}` | The link, date, reporter, weaknesss, and description of the vulnerability in the report in markdown format. |
```
Comment (saida) [inline docs/programs/integration-variables.md]:
```suggestion
`{{details_markdown}}` | The link, date, reporter, weakness, and description of the vulnerability in the report in markdown format. |
```

File: docs/programs/integration-variables.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+`{{details_truncated}}` | The link, date, reporter, weaknesss, and description of the vulnerability in the report in plain text format truncated at 4,000 characters.
```
Comment (saida) [inline docs/programs/integration-variables.md]:
```suggestion
`{{details_truncated}}` | The link, date, reporter, weakness, and description of the vulnerability in the report in plain text format truncated at 4,000 characters. 
```

File: docs/programs/integration-variables.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+Variable | Description | Example
+-------- | ------ |
+`{{assignee}}` | The user that's assigned to the report. |
+`{{days_after_report(days)}}` | Provides a date based on the report creation date and the provided arugment on the provided argument. `{{days_after_report(5)}}` will output a date 5 days after the report was created. |
+`{{details_custom_jira}}` | The link, date, reporter, weaknesss, and description of the vulnerability in the report in a special Jira flavored markdown format. |
+`{{details_markdown}}` | The link, date, reporter, weaknesss, and description of the vulnerability in the report in markdown format. |
+`{{details_truncated}}` | The link, date, reporter, weaknesss, and description of the vulnerability in the report in plain text format truncated at 4,000 characters. 
+`{{details}}` | The link, date, reporter, weaknesss, and description of the vulnerability in the report in plain text format. |
```
Comment (saida) [inline docs/programs/integration-variables.md]:
```suggestion
`{{details}}` | The link, date, reporter, weakness, and description of the vulnerability in the report in plain text format. |
```

File: docs/programs/integration-variables.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+`{{due_date(days_from_now)}}` | Provides a date in the future based on the provided argument. `{{due_date(5)}}` will output a date 5 days from now. |
+`{{email_local_part}}` | The local part of the user creating the Jira issue. The local part of an email address is the part before the @ symbol.
```
Comment (saida) [inline docs/programs/integration-variables.md]:
```suggestion
`{{email_local_part}}` | The local part of the email address of the user creating the Jira issue. The local part of an email address is the part before the @ symbol.
```

File: docs/programs/integration-variables.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+`{{email}}` | The email address of the user creating the Jira issue |
+`{{escalation_time}}` | The date and time of when the report was escalated to Jira. |
+`{{jira_severity}}` | The severity of the report in Jira mapping. `none` converts to `1` and `critical` converts to `5`. |
+`{{report_id}}` | The ID of the report |
+`{{report_link}}` | The url of the report |
```
Comment (saida) [inline docs/programs/integration-variables.md]:
```suggestion
`{{report_link}}` | The URL of the report |
```

File: docs/programs/integration-variables.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+`{{reporter_link}}` | The url of the researcher's HackerOne profile |
```
Comment (saida) [inline docs/programs/integration-variables.md]:
```suggestion
`{{reporter_link}}` | The URL of the researcher's HackerOne profile |
```

File: docs/programs/integration-variables.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+`{{reporter_name}}` | The name of the researcher that reported the vulnerability (this will fallback to the username if the researcher didn't provide their name) |
+`{{reporter_username}}` | The username of the researcher that reported the vulnerability |
+`{{submission_date_custom_format}}` | The created date of the report in `DD-MM-YYYY` |
+`{{submission_date_y_m_d}}` | The creation date of the report in `YYYY-MM-DD` |
```
Comment (saida) [inline docs/programs/integration-variables.md]:
```suggestion
`{{submission_date_y_m_d}}` | The created date of the report in `YYYY-MM-DD` |
```
To keep it consistent with the other lines.

**Pull request merged**